sphinxcontrib/needs/roles/need_value.py

```diff
@@ -0,0 +1,110 @@
+from docutils import nodes
+
+try:
+    from sphinx.errors import NoUri  # Sphinx 3.0
+except ImportError:
+    from sphinx.environment import NoUri  # Sphinx < 3.0
+from sphinx.util.nodes import make_refnode
+
+from sphinxcontrib.needs.logging import get_logger
+
+log = get_logger(__name__)
+
+
+class NeedValue(nodes.Inline, nodes.Element):
+    pass
+
+
+def process_need_value(app, doctree, fromdocname):
```

Comment:
@danwos I reused the implementation of the role need_ref. Could we maybe define a more universal function to handle this. Even it would be nice to have in need_ref access to self define options. This is here implemented.